RealTimeFaceMaskDetector/TCPServer/TCPServer.vcxproj

```diff
@@ -23,7 +23,7 @@
     <Keyword>Win32Proj</Keyword>
     <ProjectGuid>{5aec37d4-cf31-4a4c-8f6b-d29f49a203c1}</ProjectGuid>
     <RootNamespace>TCPServer</RootNamespace>
-    <WindowsTargetPlatformVersion>10.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>10.0.18362.0</WindowsTargetPlatformVersion>
```

Comment:
Hope this will not crash our application))))

Reply:
Whoops... I will change it back to 10.0 then